### ISSUE TITLE
Feature: Allow setting CLI flags via environmental variables

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1177]
ref: 

https://github.com/open-wa/wa-automate-deploy-heroku/issues/3

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.9](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.8...3.1.9) (2020-12-13)

- Release 3.1.9 [`cf1d58e`](https://github.com/open-wa/wa-automate-nodejs/commit/cf1d58eace82bf6e997850ab0be7fafe8784d07e)
- docs: try out new CHANGELOG format `news-fragments` [`f57c456`](https://github.com/open-wa/wa-automate-nodejs/commit/f57c4565aee8e1a919e40f2df730b02abb176612)
- chore: Improved listener and onAck documentation #1173 [`4c19ad4`](https://github.com/open-wa/wa-automate-nodejs/commit/4c19ad4977c7370e97a552c443ec9589e5136980)
- Feature: Implemented `listWebhooks` and `removeWebhook` #1175 [`0a26e05`](https://github.com/open-wa/wa-automate-nodejs/commit/0a26e058c25920d4de0914ded418aa644c60fa1d)
- Feature: Now able to set CLI flags via `process.env` by converting flags to camelCase then prepending `WA_` #1177 [`a38264f`](https://github.com/open-wa/wa-automate-nodejs/commit/a38264f5dbb11baab7d4572e1256bc39f765d925)
- Feature: Implemented #954 [`72f285d`](https://github.com/open-wa/wa-automate-nodejs/commit/72f285d9fe409698b2b4171fafb24597253d11ee)
- fix: system now waits for `Debug.Version` to not be `undefined` before attempting to authenticate #1168 [`88c9fc5`](https://github.com/open-wa/wa-automate-nodejs/commit/88c9fc5e01655412b6209fc5a1f0ef976204d9be)
- chore: [popup] Updated `CONNECTION CLOSED` message to be less confusing. #1176 [`4698a1a`](https://github.com/open-wa/wa-automate-nodejs/commit/4698a1abb45380fe41ec607b33c6ba4430f2ce19)
- chore: update patches [`6ff03ca`](https://github.com/open-wa/wa-automate-nodejs/commit/6ff03caa00fddd78f96ba9371deff86ac61ff817)
- patch update [`32487d3`](https://github.com/open-wa/wa-automate-nodejs/commit/32487d3572bd6e325b403eae2fc3a12296632181)

**Comment by smashah** [comment]
Implemented in the latest version: https://github.com/open-wa/wa-automate-nodejs/releases/tag/3.1.9

Now you can set CLI flags via ENV VARs. All you need to do is convert to `CONSTANT_CASE` and add a `WA_` before the variable name. Please note this will not work with shorthand aliases.

So `--keep-alive` becomes `WA_KEEP_ALIVE=true`.

As mentioned, this won't work on shorthand. For the above example, `-a` is the shorthand for `--keep-alive`, but this does not mean `WA_A=true` will work.

🆗 👍🏼   `--keep-alive` => `WA_KEEP_ALIVE=true`
🚫 🙅🏼    `--a` => `WA_A=true`

For example:

```bash
	> npx @open-wa/wa-automate --disable-spins --webhook "https://webhook.url/api"
```

is the same as

```bash
	> WA_DISABLE_SPINS=true WA_WEBHOOK="https://webhook.url/api" npx @open-wa/wa-automate
```
